go/vt/vtctl/workflow/materializer_test.go

```diff
@@ -2541,20 +2541,6 @@ func TestCreateLookupVindexFailures(t *testing.T) {
 		schemaAdditions        []*tabletmanagerdatapb.TableDefinition
 		err                    string
 	}{
-		{
-			description: "dup vindex",
-			input: &vschemapb.Keyspace{
-				Vindexes: map[string]*vschemapb.Vindex{
-					"v1": {
-						Type: "xxhash",
-					},
-					"v2": {
-						Type: "xxhash",
-					},
-				},
-			},
-			err: "only one vindex must be specified",
```

Comment:
Since we will support backfilling multiple lookup tables, this unit test can be removed.